tools/block-generator/generator/server_test.go

```diff
@@ -45,57 +45,78 @@ func TestInitConfigFileNotExist(t *testing.T) {
 	}
 }
 
-func TestParseRound(t *testing.T) {
+func TestParseURL(t *testing.T) {
 	partitiontest.PartitionTest(t)
+	const blockQueryPrefix = "http://v2/blocks/"
+	const accountQueryPrefix = "http://v2/accounts/"
+	const deltaQueryPrefix = "http://v2/deltas/"
 	var testcases = []struct {
 		name          string
 		url           string
-		expectedRound uint64
+		expectedRound string
```

Comment:
```suggestion
		expectedParam string
```

Reply:
Or maybe `expectedPart` ? No big deal regardless.

Reply:
I'll update this in the next PR.